recipe/meta.yaml

```diff
@@ -25,9 +25,7 @@ requirements:
   run:
     - python >=3.9
     - krtc
-    - six
     - requests
-    - openpyxl
```

Comment:
CI is failing on an import error without this dependency

Reply:
Looks like the test command for this is never run upstream- the upstream repo has a bad dependency set
https://github.com/pcdshub/psdm_qs_cli/blob/9d6bed253aa3a9f2c49f2fe43d3402b8cd9a6042/psdm_qs_cli/QSGenerateExcelSpreadSheet.py#L12

Reply:
hooray ci